examples/molerat/1_downsample_source_images.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+from brainglobe_space import AnatomicalSpace
+from brainglobe_utils.IO.image import load_any, save_any
+from loguru import logger
+from skimage import transform
+
+from brainglobe_template_builder.plots import plot_grid, plot_orthographic
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Downsample source images")
+    parser.add_argument(
+        "--template_building_root",
+        type=str,
+        help="Path to the template-building root folder. Results will "
+        "be written to the rawdata subfolder.",
+        required=True,
+    )
+    parser.add_argument(
+        "--target_isotropic_resolution",
+        type=int,
+        help="Target isotropic resolution",
+        required=True,
+    )
+
+    args = parser.parse_args()
+
+    atlas_forge_molerat_path = Path(
+        "/media/ceph/neuroinformatics/neuroinformatics/atlas-forge/MoleRat/"
+    )
+    source_data = atlas_forge_molerat_path / "new_d06"
+    source_info_file = atlas_forge_molerat_path / "molerat_brains_info_PM.csv"
+
+    template_building_root = Path(args.template_building_root)
```

Comment:
I'm slightly confused by sth here: is the `template_building_root` the same as `atlas_forge_molerat_path`, which is already hard-coded here? Why does it have to be supplied as an extra CLI argument?